iss_forecasting/analysis/utils/plotting.py

```diff
@@ -89,7 +90,9 @@ def lagplot(
     """
     x_ = x.shift(lag)
     y_ = y if y is not None else x
-    corr = y_.corr(x_)
+    pg_corr = pg.corr(x_, y_).round(2)
```

Comment:
Which correlation coefficient method do you think would work best here?
Looks like the default is `pearson`, which however has a normality assumption.

Reply:
Thanks for spotting this. I have changed the method to `spearman`.